### ISSUE TITLE
📎 Change ci linting solution 

### ISSUE THREAD

**Issue by jer3m01** [rome/tools#1356]
When a linting fix is found by `rome ci` the command is `rome check [...] --format`

Can be seen here https://github.com/rome/tools/pull/1354/checks?check_run_id=1882245610#step:4:49.

Should be `--apply`

**Comment by sebmck** [comment]
It's actually meant to be `--format-only`. `--apply` isn't the right flag here as the suggestion is "To format this file without any fixes". Not sure why that's appearing there though as it should be alongside the above diagnostic.

**Comment by sebmck** [comment]
Also we only show the `--format-only` suggestion when using the `--verbose-diagnostics` flag. It has `hidden: true` set which hides it during a normal `rome check` run. `rome ci` just has an implicit `--verbose-diagnostics` flag as verbosity is good for CI logs. It's also visible when using `--review` to select an action.

**Comment by jamiebuilds** [comment]
I can't see the referenced logs, but if its a lint error, why would you suggest `--format-only` to run locally?

**Comment by jamiebuilds** [comment]
Hi! Thank you for opening this issue

We recently made the decision to [migrate the Rome codebase to Rust](https://rome.tools/blog/2021/09/21/rome-will-be-rewritten-in-rust) and making some significant changes to its architecture. We've also decided not to release the previous version written in TypeScript.

We're going through and closing pull requests and issues that were specific to the TypeScript codebase, including this one. Thank you for taking the time to open it.

Thanks again!
